docs/media/dot-net-cross-platform.svg

```diff
@@ -0,0 +1,63 @@
+<path d="M108.21 145.39V147.49C108.21 148.22 108.73 148.83 109.45 148.96L175.98 160.34C177.78 160.64 181.37 160.93 183.22 160.9L343 158.74L342.81 155.18L126.64 145.93L108.21 145.39Z" fill="#9C9894"/>
+<path d="M108.21 145.13L177.35 156.11C179.87 156.51 182.46 156.69 185.04 156.67L343.56 155.1L266.49 144.64L108.21 145.13Z" fill="#CCCAC7"/>
+<path d="M396.99 96.68L225.98 90.72C221.96 90.58 218.68 94.07 218.68 98.51V223.5V224.99V239.51C218.68 243.31 221.49 246.19 224.94 245.92L397.01 234.49C399.65 234.29 401.78 231.45 401.78 228.14V215.52L402.56 103.87C402.56 100.01 400.07 96.79 396.99 96.68Z" fill="#B3AFAB"/>
+<path d="M294.18 235.94L290.18 257.88C289.8 262.6 293.37 266.79 297.96 268.01L310.16 272.97C315.07 274.45 319.67 274.66 323.82 274.16L362.83 269.82C366.76 269.65 366.48 265.98 361.31 265.51L336.91 262.68C333.07 262.25 329.45 259.21 330.17 255.15L334.17 231.93L294.18 235.94Z" fill="#BBB8B4"/>
+<path d="M404.35 215.52L405.13 103.87C405.13 100.01 402.64 96.79 399.56 96.69L228.55 90.73C224.53 90.59 221.25 94.08 221.25 98.52V225C221.25 225.64 221.72 226.12 222.3 226.09L403.56 216.58C403.99 216.54 404.35 216.07 404.35 215.52Z" fill="#625F5B"/>
+<path d="M221.24 223.5V239.52C221.24 243.33 224.05 246.2 227.5 245.93L399.57 234.5C402.21 234.3 404.34 231.46 404.34 228.15V214.23L221.24 223.5Z" fill="#DDDBDA"/>
+<path d="M322.04 226.88C322.04 228.91 320.64 230.66 318.91 230.78C317.18 230.9 315.77 229.34 315.77 227.3C315.77 225.26 317.18 223.51 318.91 223.4C320.65 223.29 322.04 224.85 322.04 226.88Z" fill="#C8C6C4"/>
+<path d="M396.64 207.75L230.35 215.08V102.17L397.42 106.63L396.64 207.75Z" fill="#8DBCF4"/>
+<path d="M397.42 105.86L230.35 101.29V105.67L397.42 109.71V105.86Z" fill="#C7E0F4"/>
+<path d="M365.62 199.74L267.86 203.43C267.05 203.47 266.39 204.25 266.39 205.17V212.88L366.89 208.37V201.23C366.89 200.37 366.33 199.71 365.62 199.74Z" fill="#3085E9"/>
+<path d="M295.68 209.5L292.41 209.67C292.03 209.69 291.73 209.35 291.73 208.92V205.13C291.73 204.7 292.03 204.33 292.41 204.31L295.68 204.15C296.05 204.13 296.36 204.47 296.36 204.9V208.67C296.35 209.11 296.05 209.48 295.68 209.5Z" fill="#8DBCF4"/>
+<path d="M286.72 207.39C286.72 208.96 285.61 210.3 284.23 210.37C282.85 210.44 281.73 209.22 281.73 207.64C281.73 206.06 282.85 204.72 284.23 204.66C285.6 204.59 286.72 205.81 286.72 207.39Z" fill="#8DBCF4"/>
+<path d="M306.34 206.26C306.34 207.81 305.26 209.13 303.93 209.2C302.59 209.27 301.51 208.06 301.51 206.51C301.51 204.96 302.6 203.64 303.93 203.57C305.26 203.5 306.34 204.71 306.34 206.26Z" fill="#8DBCF4"/>
+<path d="M316.26 205.9C316.26 207.44 315.2 208.74 313.88 208.81C312.56 208.88 311.49 207.68 311.49 206.14C311.49 204.6 312.56 203.29 313.88 203.23C315.19 203.16 316.26 204.36 316.26 205.9Z" fill="#8DBCF4"/>
```

Comment:
This line looks wrong.

Reply:
What? You find it unlikely that I could decipher SVG markup in my head? 😉

Reply:
How so? It came from the .NET brand repo, so if it's wrong here - it's wrong there as well. And the rendering looked good, no?